crates/client/gateway/src/server/service.rs

```diff
@@ -11,18 +11,24 @@ use hyper::{
 };
 use mc_db::MadaraBackend;
 use mc_rpc::providers::AddTransactionProvider;
+use mp_block::H160;
 use mp_utils::graceful_shutdown;
+use starknet_core::types::Felt;
 use tokio::net::TcpListener;
 
 use super::router::main_router;
 
+#[allow(clippy::too_many_arguments)]
 pub async fn start_server(
     db_backend: Arc<MadaraBackend>,
     add_transaction_provider: Arc<dyn AddTransactionProvider>,
     feeder_gateway_enable: bool,
     gateway_enable: bool,
     gateway_external: bool,
     gateway_port: u16,
+    eth_core_contract_address: H160,
```

Comment:
the chain config is already in MadaraBackend
Remove this

Reply:
Done